training/culture-and-process/PULL_REQUEST_GUIDANCE.md

```diff
@@ -0,0 +1,35 @@
+Pull Request Guidance
+===========================
+
+General guidance on creating a pull request
+
+### Pre reading
+
+- It is useful to have an understanding of GIT (https://git-scm.com/about) and branches as a way of working
+- Have a look at pull requests in the `dev-code-review` channel on slack
+
+### Pre requisites
+
+None
+
+### Steps
+
+To merge a feature branch into the develop or master/main branch into has to have an approved pull request. 
+
+A pull request can be created in a couple of ways, once the chnages on a local feature branch have been commited and pushed. 
+- In the terminal after pushing a branch a link appears to create a pull request. Follow this url, to create a pull request, but just check the branch (either develop or master/main) that the pull request is being created for.
+- Using the GitHub UI, navigate to the repository and the pull requests tab and use the `new pull request` button on the top right of the screen. 
+
+When creating a pull request the following structure is used to provide information to the reviewer:
+- What: A description of the changes and why
+- How to review: If required steps to enable the reviewer to do a code review
+- Who can review
+
+Once a pull request has been created put a link to the pull request in the `dev-code-review` channel on slack. At the top of the channel are the reactions used to communicate when the pull request is being reviewed, if there are any comments and when it has been approved. 
+
```

Comment:
It would be nice to mention that "re-request review " is needed , after we address any comments raised on the PR.

We could also mention about "draft PR", which can be created to raise discussions among dev's about that PR